src/gui/gui-hotlist.c

```diff
@@ -474,6 +474,86 @@ gui_hotlist_resort ()
     gui_hotlist_changed_signal ();
 }
 
+struct t_gui_hotlist *
+gui_hotlist_get ()
+{
+    struct t_gui_hotlist *ptr_hotlist, *selected;
+
+    selected = gui_hotlist;
+
+    switch (CONFIG_INTEGER(config_look_hotlist_get_sort))
+    {
+        case CONFIG_LOOK_HOTLIST_GET_SORT_GROUP_TIME_ASC:
+            for (ptr_hotlist = gui_hotlist; ptr_hotlist;
+                 ptr_hotlist = ptr_hotlist->next_hotlist)
+            {
+                if ((ptr_hotlist->priority > selected->priority)
+                    || ((ptr_hotlist->priority == selected->priority)
+                        && (util_timeval_diff (&(ptr_hotlist->creation_time),
+                                               &(selected->creation_time)) > 0)))
+                    selected = ptr_hotlist;
+            }
+            return selected;
+            break;
```

Comment:
These `break`s are superfluous when already `return`ing.

Reply:
I thought I saw some return + breaks somewhere and that it was a styling convention to have break in every case. I'll take them out.